File: seed/swift-sdk/pagination/custom-pager/src/Pagination/Users/Page.swift
public struct Page: Codable, Hashable {
    public let page: Int
    public let next: NextPage?
    public let perPage: Int
    public let totalPage: Int

    enum CodingKeys: String, CodingKey {
        case page
        case next
        case perPage = "per_page"
        case totalPage = "total_page"
    }
}

File: seed/ruby-sdk-v2/pagination/lib/seed/users/page.rb
# frozen_string_literal: true

module Seed
    module Types
        class Page < Internal::Types::Model
            field :page, Integer, optional: false, nullable: false
            field :next_, Seed::Users::NextPage, optional: true, nullable: false
            field :per_page, Integer, optional: false, nullable: false
            field :total_page, Integer, optional: false, nullable: false

    end
end
